<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QWidget" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>130</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>72</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Wynajem z klasą</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item alignment="">
      <widget class="QPushButton" name="openMainWindowButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#openMainWindowButton {
    background-color: #007FFF;
    border: none;
    border-radius: 10px; 
    font-size: 22px;  
}

#openMainWindowButton:hover {
    background-color: #004c9e;
}
</string>
       </property>
       <property name="text">
        <string>Wyszukaj Pojazd</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item alignment="">
      <widget class="QPushButton" name="openAllReservationsButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#openAllReservationsButton {
    background-color: #007FFF;
    border: none;
    border-radius: 10px; 
    font-size: 22px;  
}

#openAllReservationsButton:hover {
    background-color: #004c9e;
}
</string>
       </property>
       <property name="text">
        <string>Twoje Rezerwacje</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item alignment="">
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#closeButton {
    background-color: #007FFF;
    border: none;
    border-radius: 10px; 
    font-size: 22px;  
}

#closeButton:hover {
    background-color: #004c9e;
}
</string>
       </property>
       <property name="text">
        <string>Wyjście</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
